src/NECompletion/NECContext.class.st

```diff
@@ -395,7 +406,8 @@ NECContext >> initialize [
 	selectors := true.
 	ranges := OrderedCollection new.
 	completionIndex := 0.
-	completionToken := nil
+	completionToken := nil.
+	globals := Smalltalk.
```

Comment:
I think it is better to use `self class environment` instead of Smalltalk directly.

Someone told me that one time, like that we could have other environments than Smalltalk in the future.

Reply:
Maybe you can ask confirmation to Guille.

Reply:
Yes